news/2017-07-22-beatmap-spotlight-june-2017.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Beatmap Spotlights: June 2017"
+permalink: 
+date: 2017-07-01 14:50:00 +0000
+tumblr_url: 
+---
+
+
+Demetori, Infected Mushroom, Seiryu and KRUX reign supreme in the [Beatmap Spotlights](https://osu.ppy.sh/p/chart?ch=MONTH1705) of June 2017, and more to boot! Come check out what's in store this month, and maybe even win an osu!supporter tag for your trouble!
```

Comment:
Should be `MONTH1706`. Also maybe come up with a different intro sentence and don't recycle the same from every month.